internal/pkg/template/manifest/manifest_test.go

```diff
@@ -96,15 +81,6 @@ func minimalRecords() []model.ObjectManifest {
 
 func fullJsonNet() string {
 	return `{
-  naming: {
-    config: "{component_type}/{component_id}/{config_id}",
-    configRow: "rows/{config_row_id}",
-    schedulerConfig: "schedules/{config_id}",
-    sharedCodeConfig: "_shared/{target_component_id}",
-    sharedCodeConfigRow: "codes/{config_row_id}",
-    variablesConfig: "variables",
-    variablesValuesRow: "values/{config_row_id}",
-  },
```

Comment:
Z template `manifest.jsonnet` som vyhodil naming.
Narozdiel od projektu nema velmi zmysel ho menit.
Tak bude napevno, a bude to prehladnejsie pre autora sablony (ze v manifeste su iba objects).